user-guide/Advanced_Modules/DOM/DOM_objects/DOM_FieldDescriptors/DOM_ServiceDefinitionFieldDescriptor.md

```diff
@@ -0,0 +1,29 @@
+---
+uid: DOM_ServiceDefinitionFieldDescriptor
+---
+
+# ServiceDefinitionFieldDescriptor
+
+- **FieldValue type**: `GUID` of an SRM Resource
+- **Multiple values supported**: :heavy_check_mark:
+
+| Type of Descriptor | FieldType | FieldValue type |
+|--------------------|-----------|-----------------|
+| ServiceDefinition | Guid | Guid |
+| ServiceDefinition with multiple values enabled| List\<Guid\> | Guid (ListValueWrapper) |
+
+Defines a field that has the `GUID` of an SRM `ServiceDefinition`. It contains a `ServiceDefinitionFilter` property that has a `FilterElement` that can be used to determine which service definitions will be presented to the user.
```

Comment:
service definitions -> `ServiceDefinitions` to be consistent with the line below.